### ISSUE TITLE
Secure FPP / FPP Connect doesnt work first time and Save Password doesnt work

### ISSUE THREAD

**Issue by Fonzi03** [xLightsSequencer/xLights#4977]
2024.18

Have a password protected FPP instance (FPPOS 7.5 PI4)

- FPP Connect
- Type Password and select Save Password
- Select OK
When the FPP Upload window appears there are no detected FPP instances. 
Cancel and try again, (saved password works) Discovering FPP window appears and when FPP Upload loads the FPP instance is shown. 

Next time you open xLights and try to FPP connect the password field is requested again (even though save password was selected previously)

The same issue with not detecting (or attempting to discover FPP) happens and you must type the password, and retry FPP connect. 

100% reproducibility

**Comment by AGFazio** [comment]
Don't use the password option. It causes issues when uploading. Set UI passsword to disabled.

**Comment by Fonzi03** [comment]
> Don't use the password option. It causes issues when uploading. Set UI passsword to disabled.
> 

Yeah not going to do that since it's exposed to the WAN. Does it have any other issues than what I described? If not I'll deal with it until it's fixed.

**Comment by cybercop23** [comment]
it is fixed, but doesn't really work. Why is it exposed? How are you using it?

**Comment by derwin12** [comment]
Can you please retest with newer versions. FPP is now at version 9 and many changes have taken place since this was created.
Please create a new ticket with versions tested if it is still an issue.
